cbrain_task/civet_macaque/bourreau/civet_macaque.rb

```diff
@@ -447,6 +447,9 @@ def save_results
     # Where we find this subject's results
     out_dsid = "civet_out/#{dsid}"
 
+    # By default saved to CivetOutput
+    saved_to_class = CivetOutput
```

Comment:
Instead of having this variable maintain the state of the failure as a class name, why don't you use directly

```ruby
    is_partial = false
```

which is a variable that you derived out of the class name anyway later on?